containerz/containerz.proto

```diff
@@ -629,14 +634,17 @@ message CreateVolumeRequest {
   // and associated options.
   Driver driver = 2;
 
-  // Options for the driver. The actual option keys and values are driver
+  reserved 3;
+
+  // Labels to apply to the volume. Labels are metadata for the the volume.
+  map<string, string> labels = 10;
+
+  // Options for the driver. The actu:al option keys and values are driver
   // specific.
   oneof options {
-    LocalDriverOptions local_mount_options = 3;
+    LocalDriverOptions local_mount_options = 20;
+    CustomOptions custom_options = 21;
   }
-
-  // Labels to apply to the volume. Labels are metadata for the the volume.
-  map<string, string> labels = 4;
```

Comment:
Any reason that this changed field ID but looks to have stayed in the same place in the proto?

Reply:
Well I wanted to create some room in the proto for future fields and options so I figured this is a better time to pay this price rather that later when this is more in use.

Reply:
Not sure I understand -- we're making a backwards incompatible change, but I don't understand why. Maybe missing something.

If we have `labels` continue to use tag 3, it's not like we can't add other fields later on elsewhere -- the same is true for the tag change for `local_mount_options`. The tags don't have to be sequential in the proto.

Reply:
Unfortunately I had sandwiched options between driver (2) and labels (4). So I can either do what I did in this PR or other options would have to start at 5 (or more). Happy to do either I just opted for the one which would be less error prone going forward.

Reply:
IMHO, we should bias towards backwards compatibility as soon as these things are committed and consumed. Any proto that evolves ends up having some of these out of order  field tag issues.

Reply:
True - I restablished the numbering.